lib/krikri/entity_behavior.rb

```diff
@@ -0,0 +1,18 @@
+
+module Krikri
+  ##
+  # Base class for behaviors related to entities that are generated by
+  # activities.
+  class EntityBehavior
```

Comment:
This could do with method/attribute documentation, and probably an `@example` in the class level doc.

Consider using: http://www.rubydoc.info/gems/yard/file/docs/Tags.md#abstract